uwblueprint/planet-read
# Add date_uploaded column to stories

## Notion ticket link



[Add date_uploaded column on stories table](https://www.notion.so/uwblueprintexecs/Add-date_uploaded-column-on-stories-table-7b376c129ac84177a67829dd734cf44f)



## Implementation description

- add date_uploaded (datetime) column to stories
- populate with utc current time in create_story 



## Steps to test

1. `docker exec -it planet-read_py-backend_1 /bin/bash -c "python -m tools.insert_test_data erase"`
2. `planet-read % docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db upgrade"`
3. `docker exec -it planet-read_py-backend_1 /bin/bash -c "python -m tools.insert_test_data"`
4. [Try creating a story](http://localhost:5000/graphql?variables=null&operationName=createStory&query=mutation%20createStory%20%7B%0A%20%20createStory(contents%3A%5B%22once%22%2C%20%22upon%22%2C%20%22a%22%2C%20%22time%22%5D%2C%20storyData%3A%7Btitle%3A%22title%22%2C%20description%3A%22description%22%2C%20youtubeLink%3A%22google.ca%22%2C%20level%3A%201%7D)%7B%0A%20%20%20%20ok%0A%20%20%7D%0A%7D) - check that the time col is added 



## What should reviewers focus on?

- Does it work?

## Checklist
